ibm/service/vpc/resource_ibm_is_subnet_routing_table_attachment.go

```diff
@@ -233,6 +255,30 @@ func resourceIBMISSubnetRoutingTableAttachmentUpdate(context context.Context, d
 		return resourceIBMISSubnetRoutingTableAttachmentRead(context, d, meta)
 	}
 
+	if d.HasChange(isRoutingTableCrn) {
```

Comment:
verify if this handles the case where user removes crn and adds id, or vice versa

Reply:
```
resource "ibm_is_subnet_routing_table_attachment" "attach" {
  # depends_on    = [ibm_is_vpc_routing_table.testacc_vpc_routing_table, ibm_is_subnet.testacc_subnet]
  subnet = "0726-9d4bd36c-5d9f-4494-beed-15d5ed0f021d"
  routing_table   = ibm_is_vpc.testacc_vpc.default_routing_table
  # routing_table = ibm_is_vpc_routing_table.test_route_rt.routing_table
  # routing_table_crn = ibm_is_vpc_routing_table.test_route_rt.crn
}
```
<img width="1108" alt="Screenshot 2024-08-04 at 8 32 16 PM" src="https://github.com/user-attachments/assets/da798567-d032-4981-bd92-007c1e8270b7">

Reply:
same routing table/crn no diff, different id or crn will result into force replace.